File: dayTwo/JAVA/LEVEL ONE/TaskTen.java
import java.util.Scanner;
public class TaskTen{
    public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.print("Enter first number: ");   
    double numberOne = input.nextDouble();
    
    System.out.print("Enter second number: "); 
    double numberTwo = input.nextDouble();

    System.out.print("Enter third number: "); 
    double numberThree = input.nextDouble();    
     
    double average = (numberOne + numberTwo + numberThree) / 3;
    System.out.printf("Average: %.2f%n", average);   
    }
}

File: dayTwo/JAVASCRIPT/LEVEL ONE/taskten.js
const prompt = require("prompt-sync")()

let numberOne = Number(prompt("Enter first number: "));
let numberTwo = Number(prompt("Enter second number: "));
let numberThree = Number(prompt("Enter third number: "));

const average = (numberOne + numberTwo + numberThree) / 3;

console.log("Average", average.toFixed(2))
